client/gutenberg/editor/index.js

```diff
@@ -56,6 +56,11 @@ export default function() {
 				clientRender
 			);
 		}
+
+		page( '/block-editor/**/*', '/block-editor/post' );
```

Comment:
What's the point of double asterisk here? I'd expect this to work with `/block-editor/*/*`.

Reply:
No point at all, not sure why I did that 🙃 Removed it 👍